cmd/fabric8-jenkins-proxy/main.go

```diff
@@ -238,6 +265,17 @@ func newAPIServer(api api.ProxyAPI) *http.Server {
 	return srv
 }
 
+func newStatusAPIServer(statusapi statusapi.JenkinsStatusAPI) *http.Server {
+	c := cors.New(cors.Options{
+		AllowCredentials: true,
+	})
+	srv := &http.Server{
+		Addr:    jenkinsStatusAPIRouterPort,
+		Handler: c.Handler(router.CreateJenkinsStatusAPIRouter(statusapi)),
```

Comment:
It may be better to create a middleware that does the authentication so that all api need not worry about auth as it will be taken care by the middleware.